packages/esbuild/esbuild_repositories.bzl

```diff
@@ -0,0 +1,108 @@
+"""
+Helper macro for fetching esbuild versions for internal tests and examples in rules_nodejs
+"""
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@build_bazel_rules_nodejs//internal/common:check_version.bzl", "parse_version")
+
+_VERSION = "0.12.1"
+_MIN_VERSION = "0.12.0"
+_MAX_VERSION = "0.13.0"
+
+DARWIN_AMD64 = "darwin_amd64"
```

Comment:
what about darwin_arm64? maybe a note here about what platforms are supported by esbuild and how we chose the list to mirror?

Reply:
Yes, see comment in this diff

Reply:
Done, added 🦾